packages/frontend/src/components/DashboardFilter/FilterConfiguration/index.tsx

```diff
@@ -125,6 +125,8 @@ const FilterConfiguration: FC<Props> = ({
                 createDashboardFilterRuleFromField(
                     newField,
                     availableTileFilters,
+                    false,
+                    isCreatingNew && !isEditMode,
```

Comment:
I would even have a variable for this: 
```suggestion
                 const isCreatingNewInViewMode =  isCreatingNew && !isEditMode,
```

Reply:
👍

Reply:
Done